src/frame/worker/tests/GroupCrypto.test.ts

```diff
@@ -100,33 +100,37 @@ describe("GroupCrypto", () => {
 
     describe("addAdminsToGroup", () => {
         it("decrypts group private key encrypts it to the provided list of public keys", () => {
-            spyOn(Recrypt, "decryptPlaintext").and.returnValue(Future.of(["decryptedPlaintext"]));
-            spyOn(Recrypt, "encryptPlaintextToList").and.returnValue(Future.of(["accessKey1", "accessKey2"]));
+            const signature = new Uint8Array(32);
+            const groupPublicKey = TestUtils.getEmptyPublicKeyString();
+            jest.spyOn(Recrypt, "decryptPlaintext").mockReturnValue(Future.of(["decryptedPlaintext"]) as any);
+            jest.spyOn(Recrypt, "encryptPlaintextToList").mockReturnValue(Future.of(["accessKey1", "accessKey2"]) as any);
+            jest.spyOn(Recrypt, "schnorrSignUtf8String").mockReturnValue(signature as any);
 
             const groupPrivateKey = TestUtils.getTransformedSymmetricKey();
             const adminPrivateKey = new Uint8Array(20);
             const userList = [{id: "user-35", masterPublicKey: {x: "", y: ""}}];
             const signingKeys = TestUtils.getSigningKeyPair();
 
-            GroupCrypto.addAdminsToGroup(groupPrivateKey, userList, adminPrivateKey, signingKeys).engage(
+            GroupCrypto.addAdminsToGroup(groupPrivateKey, groupPublicKey, "grouID", userList, adminPrivateKey, signingKeys).engage(
```

Comment:
nitpick: I know it doesn't matter, but did you mean for `grouID` to be `groupID`?

Reply:
There should probably be an assertion below that this value was used in the expected place. Which probably means a spy on the `Recrypt.schnorrSignUtf8String` method.

Reply:
done